etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverter.java

```diff
@@ -26,8 +25,7 @@
  * Order <Bundle>}). Also converts an order to identify as an HL7v2 OML in the {@link
  * MessageHeader}.
  */
-public class HapiOrderConverter implements OrderConverter {
-    private static final HapiOrderConverter INSTANCE = new HapiOrderConverter();
+public class HapiOrderConverter {
```

Comment:
You changed these converters to be unable to be `@Injected` and pure helper static classes.  What is your reasoning behind this?

Reply:
This implementation changed now that we're using the transformation engine. Methods in this class are only called in custom transformations. We could actually move most of this code into the custom methods but I thought we could leave them as a helper class that could be reuse by multiple transformations